fastlane/Fastfile

```diff
@@ -0,0 +1,64 @@
+# This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
```

Comment:
Somewhere in these comments, it might be good to describe the purpose of this `Fastfile`. 
```
# This Fastfile is available to all builds that import the BuildActions plugin.
# A default implementation for generating a Veracode build is provided and made available via the 'buildVeracode' lane (including default error handling) or via 'buildVeracodeCore' (without default error handling).
```

Also, as a side note, I think it would be neat if we could define another lane in here for general purpose build creation, since the only real difference between most projects is the scheme and provisioning profile names. Definitely something for a future PR though.

Reply:
Updated. Will add an issue for adding a general purpose build lane.